subsys/mgmt/mcumgr/lib/cmd/shell_mgmt/src/shell_mgmt.c

```diff
@@ -107,10 +107,10 @@ shell_mgmt_exec(struct mgmt_ctxt *ctxt)
 	cmd_out.value = shell_get_output(&cmd_out.len);
 
 	/* Key="o"; value=<command-output> */
-	/* Key="rc"; value=<status> */
+	/* Key="ret"; value=<status> */
 	ok = zcbor_tstr_put_lit(zse, "o")		&&
 	     zcbor_tstr_encode(zse, &cmd_out)		&&
-	     zcbor_tstr_put_lit(zse, "rc")		&&
+	     zcbor_tstr_put_lit(zse, "ret")		&&
```

Comment:
Do some Kconfig for legacy  behaviour. And check how it behaves with mcumgr-cli.

Reply:
Updated. Both work with mcumgr, checked with custom tool that the Kconfig option does change the name.